File: src/main/java/com/ject/studytrip/studylog/infra/jpa/StudyLogDailyMissionJpaRepository.java
package com.ject.studytrip.studylog.infra.jpa;

import com.ject.studytrip.studylog.domain.model.StudyLogDailyMission;
import org.springframework.data.jpa.repository.JpaRepository;

public interface StudyLogDailyMissionJpaRepository
        extends JpaRepository<StudyLogDailyMission, Long> {}

File: src/main/kotlin/com/ject/studytrip/studylog/infra/jpa/StudyLogDailyMissionJpaRepository.kt
package com.ject.studytrip.studylog.infra.jpa

import com.ject.studytrip.studylog.domain.model.StudyLogDailyMission
import org.springframework.data.jpa.repository.JpaRepository

interface StudyLogDailyMissionJpaRepository : JpaRepository<StudyLogDailyMission, Long>
